src/main/java/com/thebois/models/world/World.java

```diff
@@ -175,7 +174,7 @@ private boolean isPositionPlaceable(final Position position) {
      *
      * @return the colony.
      */
-    public IBeingGroup getColony() {
+    public Colony getColony() {
```

Comment:
No simple way of changing it without implementing a new interface like `IInventory` which would be temporary.

Reply:
>No simple way of changing it without implementing a new interface like IInventory which would be temporary.

Maybe it wouldn't, since it's good to have an interface between the concrete `Inventory` and its clients?